dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
I will not release any Prisoner of **W**ar but here is the plan for DNF 2.0:

* libhif 0.7.0 will be released once DNF 2.0 is ready (for F25+)
* libhif API will stabilize in 1.0 (F26/F27)
* all changes in C code (hawkey) are made in libhif only so any DNF features requiring new code in libhif will be available just in DNF 2.0
* DNF 2.0 changes are:
- [ ] remove a few compatible Python API method wrappers
- [ ] add to all non-API methods `_` prefix
- [ ] change parsing of CLI arguments (more strict and better --help handling)
- [ ] plugins parsing API can change a bit
- [ ] change a bit --installroot
- [ ] restore `includepkgs` yum config option
- [ ] move repoquery into DNF
- [ ] replace DNF protected_packages plugin with `protected_packages` config option.
- [ ] move DNF system-upgrade to plugins core
- [ ] ?make `/etc/distro/repos.d` default repos conf directory

The master branch is the upstream branch.